pkg/operation/care/health_check_test.go

```diff
@@ -1012,15 +981,6 @@ var _ = Describe("health check", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(exitCondition).To(conditionMatcher)
 		},
-		Entry("all healthy (1.70)",
-			requiredLoggingControlPlaneDeployments,
-			requiredLoggingControlPlaneStatefulSetsLessThan171,
-			false,
-			true,
-			true,
-			BeNil(),
-			gardenerVersion170,
-		),
 		Entry("all healthy (1.71)",
```

Comment:
```suggestion
		Entry("all healthy",
```
We can drop `gardenerVersion` also from the function, right?

Reply:
good idea 😄